code/modules/events/ghost_role/changeling_event.dm

```diff
@@ -98,3 +105,22 @@
 
 /obj/effect/meteor/meaty/changeling/ram_turf()
 	return //So we don't instantly smash into our occupant upon unloading them.
+
+/obj/effect/meteor/meaty/changeling/shield_defense(obj/machinery/satellite/meteor_shield/defender)
+	landing_target = defender
+	return TRUE
+
+/obj/effect/meteor/meaty/changeling/handle_stopping() //If the meteor misses the station and deletes itself, we make absolutely sure the changeling reaches the station.
+	if(!landing_target)
+		landing_target = locate(/obj/effect/landmark/observer_start) in GLOB.landmarks_list //If our destination turf is gone for some reason, we chuck them at the observer_start landmark (usually at the center of the station) as a last resort.
```

Comment:
```suggestion
		//If our destination turf is gone for some reason, we chuck them at the observer_start landmark (usually at the center of the station) as a last resort.
		landing_target = locate(/obj/effect/landmark/observer_start) in GLOB.landmarks_list
```

looks cleaner IMO

Reply:
also, do you know if this works on multi-z stations? What happens if the observer_start landmark is on the top-z, but our meteor is on the bottom-z? can we account for that?

Reply:
honestly might be worth to ensure that we only ever spawn on the z-level of the observer-start landmark just to ensure that this diversion works? not sure at all.